projects/angular/src/progress/spinner/_properties.spinner.scss

```diff
@@ -13,8 +13,6 @@
     --clr-spinner-medium-size: #{variables.$clr_baselineRem_1_5};
     --clr-spinner-small-size: #{variables.$clr_baselineRem_0_75};
     --clr-spinner-margin-right: #{variables.$clr_baselineRem_0_5};
-    --clr-spinner-inside-button-size: #{variables.$clr_baselineRem_0_65};
-    --clr-spinner-inside-small-button-size: #{variables.$clr_baselineRem_0_5416};
```

Comment:
I would list this as a breaking change due to these variables being removed. If these are defined in any application, users should know there is no longer any effect.

Reply:
I guess it doesn't matter too much since these are new temporary tokens that we just added.

Reply:
Yes, these were introduced during the token migration, so we don't expect to break clients' work by removing them